spark_client/domain.py

```diff
@@ -157,24 +157,24 @@ def __init__(self, environ: Dict = dict(os.environ)):
         self.environ = environ if environ is not None else {}
 
     @property
-    def snap_folder(self) -> str:
+    def spark_folder(self) -> str:
         """Return the SNAP folder"""
-        return self.environ["SNAP"]
+        return self.environ.get("SNAP", self.environ.get("HOME"))
```

Comment:
Sorry, for not realizing this earlier, but it is somewhat connected to the other PR. Here we should probably decide where this env variable should be "SPARK_HOME"

Reply:
pinned to SPARK_HOME in non-snap settings.